02-Git/main.c

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Please fix all formatting issues in commits where they were introduced.
No patches to unmerged patches.